attest/eventlog.go

```diff
@@ -31,6 +31,17 @@ import (
 	"github.com/google/go-tpm/tpmutil"
 )
 
+// ReplayError describes the parsed events that failed to verify against
+// a particular PCR.
+type ReplayError struct {
+	BadEvents   []Event
```

Comment:
nit: this could probably just be "Events" right? "Bad" is already implied since it's an error.

Reply:
Done.